content/kots-cli/get/config.md

```diff
@@ -0,0 +1,29 @@
+---
+date: 2021-12-01
+linktitle: "config"
+title: kots get config
+weight: 90202
+---
+
+The `kots get config` command returns the configValues file for an application.
+
+### Usage
+
+```bash
+kubectl kots get config [flags]
+```
+
+- _Provide `[flags]` according to the table below_
+
+| Flag              | Type   | Description                                                         |
+| :---------------- | ------ | ------------------------------------------------------------------- |
+| `--appslug`       | string | app slug to retrieve config for _(required)_                        |
+| `--sequence`      | int    | app sequence to retrieve config for _(required)_                    |
+| `-h, --help`      |        | help for get apps                                                   |
```

Comment:
```suggestion
| `-h, --help`      |        | The help for the `get apps` command                                                  |
```

Reply:
not touching this, as it's shared with a bunch of other commands
fixing them all at once might be a good idea though